db/schema.rb

```diff
@@ -122,6 +122,7 @@
     t.string "password_digest"
     t.datetime "created_at", null: false
     t.datetime "updated_at", null: false
+    t.boolean "super_user"
```

Comment:
This is the updated column to help facilitate a super_user experience.

The concept is that the person initializing the application is the best positioned to benefit from being the first to sign-in and automatically have superuser functionality at the beginning.

This practice was taught to me by Spoke Community Engineers and was implemented with an ability to help make onboarding faster.

I think about the security aspect of this kind of column on our application.  Which is why it's important that there be authorizing-levels to all client-protected routes.

Superuser injection is also very practical when onboarding new team members and onboarding them.